scripts/skills/effects/rf_bone_plating_effect.nut

```diff
@@ -0,0 +1,34 @@
+this.rf_bone_plating_effect <- ::inherit("scripts/skills/skill", {
+	m = {
+		SoundOnProcced = [
+			"sounds/enemies/skeleton_hurt_01.wav",
+			"sounds/enemies/skeleton_hurt_02.wav",
+			"sounds/enemies/skeleton_hurt_03.wav",
+			"sounds/enemies/skeleton_hurt_04.wav",
+			"sounds/enemies/skeleton_hurt_06.wav"
+		]
+	},
```

Comment:
Vanilla doesn't put an empty line between `m,` and `function create()` so I suggest we keep that style.